File: sources/sushimi-console/sushimi-console/app/kz/sushimi/console/models/dictionaries/CityModel.java
package kz.sushimi.console.models.dictionaries;

import java.util.Date;

/**
 * Модель для отображения списка городов
 * @author Demart
 *
 */
public class CityModel {

	private Long id;
	private String code;
	private String name;
	private String user;
	private Date createdDate;
	private Date modifiedDate;
	
	public String getCode() {
		return code;
	}
	public void setCode(String code) {
		this.code = code;
	}
	public void setId(Long id) {
		this.id = id;
	}
	public long getId() {
		return id;
	}
	public void setId(long id) {
		this.id = id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getUser() {
		return user;
	}
	public void setUser(String user) {
		this.user = user;
	}
	public Date getCreatedDate() {
		return createdDate;
	}
	public void setCreatedDate(Date createdDate) {
		this.createdDate = createdDate;
	}
	public Date getModifiedDate() {
		return modifiedDate;
	}
	public void setModifiedDate(Date modifiedDate) {
		this.modifiedDate = modifiedDate;
	}
	
}

File: sources/sushimi-console/bin/public/extjs/app/model/CityModel.js
Ext.define('SushimiConsole.model.CityModel', {
    extend: 'Ext.data.Model',
    idProperty: 'id',
    fields: [
		{ name: 'id', type: 'int'}, 
		{ name: 'code'},
		{ name: 'name'}, 
		{ name: 'user'},
		{ name: 'createdDate', type: 'date'},
		{ name: 'modifiedDate', type: 'date'},
	],
});